install_config/configuring_vsphere.adoc

```diff
@@ -141,6 +141,11 @@ Datastore Storage Folder
 
 |===
 
+[NOTE]
```

Comment:
s/VMware official article to be aware about any change/update about needed roles./article for changes or updates to required roles.